docs/pages/pmd/languages/language_properties.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Language configuration
+permalink: pmd_languages_configuration.html
+author: Clément Fournier
+last_updated: July 2022 (7.0.0)
+tags: [languages]
+keywords: [pmd, cpd, options, command, auxclasspath, language, properties]
+summary: "Summary of language configuration options and properties"
+---
+
+# Language properties
+
+Since PMD 7.0.0, languages may be directly configured via properties.
+
+TODO describe CLI syntax
+TODO describe env var syntax
```

Comment:
Thanks for finishing the doc!

Since I expect that the mapping of environment variables to language properties will not be removed even after we can set lang properties with the CLI, I changed the casing convention to look nicer (see b0b2b35fed).